examples/metrics/src/main/java/io/opentelemetry/example/metrics/DoubleMeterExample.java

```diff
@@ -0,0 +1,22 @@
+package io.opentelemetry.example.metrics;
+
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.metrics.DoubleCounter;
+import io.opentelemetry.metrics.Meter;
+
+public class DoubleMeterExample {
+
+  public static void main(String[] args) {
+    Meter sampleMeter = OpenTelemetry.getMeterProvider().get("sample", "0.1");
+    DoubleCounter jvmUsageCounter = sampleMeter.doubleCounterBuilder("jvm_memory_usage")
```

Comment:
this is probably a bad example of how to use a counter. Counters are intended to be used as a part of a "synchronous" measurement, associated with a span context. Measuring jvm memory usage is a better use-case for an Observer instrument.

Reply:
@jkwatson maybe u have smth on mind to count wia double counter?

Reply:
I don't. I'm not sure what a use-case is for counting things that are non-integer valued. Maybe total number of kilobytes processed in some kind of web request?

Reply:
but they are also int

Reply:
you can have fractional kb, though, right?

Reply:
ok i got smth on my mind